docs/Configuration/fleet-server-configuration.md

```diff
@@ -2896,6 +2896,40 @@ The duration between DEP device syncing (fetching and setting of DEP profiles).
   mdm:
     apple_dep_sync_periodicity: 10m
   ```
+##### fleet_mdm_windows_wstep_identity_cert_bytes
```

Comment:
```suggestion
##### mdm.windows_wstep_identity_cert_bytes
```

This is a weird one but for each config option's title, we remove the `fleet` bit. Not sure why but that's the way it is.

Also, `mdm` should have a dot (`.`) after it instead of an underscore. This is to communicate that the `windows_wstep_identity_cert_bytes` option lives nested under the `mdm` section.